IFC4x3/Properties/s/SlidingLoad_2DlLMolAz7CuubrDKi_Mah/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SlidingLoad_2DlLMolAz7CuubrDKi_Mah" Name="SlidingLoad" UniqueId="8dbd55b2-bcaf-4733-8e25-d4d52cf9692b" PrimaryDataType="IfcForceMeasure">
+	<Documentation></Documentation>
```

Comment:
No Documentation

Reply:
I have erased the concept. Commit : 305806220fb93c9010687f137214b3f48df310cb